src/pytest_dbfixtures/factories/dynamodb.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (C) 2013 by Clearcode <http://clearcode.cc>
+# and associates (see AUTHORS).
+
+# This file is part of pytest-dbfixtures.
+
+# pytest-dbfixtures is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# pytest-dbfixtures is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+
+# You should have received a copy of the GNU Lesser General Public License
+# along with pytest-dbfixtures.  If not, see <http://www.gnu.org/licenses/>.
+import pytest
+
+from pytest_dbfixtures.executors import TCPExecutor
+from pytest_dbfixtures.utils import try_import, get_process_fixture
+
+
+def dynamodb_proc(jar_path=None, host=None, port=None):
+    """
+    DynamoDB process factory.
+
+    :param str jar_path: path to jar file (http://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBLocal.html?shortFooter=true)
+    :param str host: hostname
+    :param int port: port
+    :return: function which makes a DynamoDB process
+    """
+    @pytest.fixture(scope='session')
+    def dynamodb_proc_fixture(request):
+        """
+        #. Run a ``DynamoDBLocal.jar`` process.
+        #. Stop ``DynamoDBLocal.jar`` process after tests.
+
+        :param FixtureRequest request: fixture request object
+        :rtype: pytest_dbfixtures.executors.TCPExecutor
+        :returns: tcp executor
+        """
+        dynamodb_executor = TCPExecutor(
```

Comment:
http://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBLocal.html?shortFooter=true

Question: What do we want to support?